src/org/opensolaris/opengrok/history/HistoryGuru.java

```diff
@@ -341,69 +343,95 @@ public void addRepositories(File[] files, Collection<RepositoryInfo> repos,
      */
```

Comment:
When you say javadoc, do you mean adding info to the comment above the addRepositories method?